screengrab/example/build.gradle

```diff
@@ -28,18 +27,15 @@ android {
     }
 }
 
-def supportLibVersion = '25.1.0'
-
 dependencies {
-    compile "com.android.support:appcompat-v7:${supportLibVersion}"
-    compile "com.android.support:design:${supportLibVersion}"
+    implementation "com.android.support:appcompat-v7:28.0.0-rc02"
+    implementation "com.android.support:design:28.0.0-rc01"
```

Comment:
it's okay that they have a different versions starting with 28

Reply:
`rc`? no real release?